api/peripheral_api/routes/Printer.js

```diff
@@ -50,6 +49,7 @@ router.post('/sendPrintRequest', async (req, res) => {
       location: response.Location,
       fileNo: fileName,
       copies: copies,
+      pageRanges: pageRanges,
```

Comment:
doing `pageRanges: pageRanges,` implies:
```suggestion
      pageRanges,
```

your call if you wanna add it.